### PR TITLE
Add induction tutor

### PR DESCRIPTION
**Pull request by goodviber**
### Context
Trello #494 NB THIS IS ONLY FOR CREATE, NOT UPDATE AS DISCUSSED.
### Changes proposed in this pull request
Addition of admin feature to create induction tutor from school overview page. 
### Guidance to review
As we have no update feature, the first created induction tutor will remain visible in the overview page even after further induction tutors have been created
### Testing
Happy path at present, will need extending when we introduce update feature
### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

**Comment by ethanmills** [comment]
[School detail: add induction tutor](https://trello.com/c/dmFmUW4O/494-school-detail-add-induction-tutor)

**Comment by goodviber** [comment]
[School detail: add induction tutor](https://trello.com/c/dmFmUW4O/494-school-detail-add-induction-tutor)

File: spec/cypress/integration/admin/InductionCoordinatorManagment.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  Background:
+    Given I am logged in as an "admin"
+    And scenario "school_with_local_authority" has been run
+    And I am on "admin schools" page
+
+  Scenario: Create an induction tutor
+    When I click on "link" containing "Test school"
+    Then I should be on "admin school overview" page
+    And "page body" should contain "Test school"
+    And the page should be accessible
```
Comment (kicferk1) [inline spec/cypress/integration/admin/InductionCoordinatorManagment.feature:13]: Do we want some percy snapshots?

File: spec/cypress/integration/admin/SchoolManagment.feature [hunk 2]
```diff
@@ -7,7 +7,7 @@ Feature: Admin user managing schools
     And I am on "admin schools" page
 
   Scenario: Viewing a school
-    When  I click on "link" containing "Include this school"
+    When I click on "link" containing "Include this school"
```
Comment (kicferk1) [inline spec/cypress/integration/admin/SchoolManagment.feature:10]: Good spot :)

File: app/controllers/admin/schools/induction_coordinators_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Admin
+  class Schools::InductionCoordinatorsController < Admin::BaseController
+    skip_after_action :verify_authorized
+    skip_after_action :verify_policy_scoped
```
[the remark this answers is not in the document]
Comment (ethanmills) [reply app/controllers/admin/schools/induction_coordinators_controller.rb:6]: I don't think we _should_ be skipping here. I think this should be authorised?

[the remark this answers is not in the document]
Comment (kicferk1) [reply app/controllers/admin/schools/induction_coordinators_controller.rb:6]: Should it? The parent controller makes sure we are an admin before every action, right?

[the remark this answers is not in the document]
Comment (goodviber) [reply app/controllers/admin/schools/induction_coordinators_controller.rb:6]: I think the parent controller requires admin, as I read it

File: spec/cypress/integration/admin/InductionCoordinatorManagment.feature (hunk 1 above)
Comment (goodviber) [reply spec/cypress/integration/admin/InductionCoordinatorManagment.feature:13]: When I put one in, the response was "Percy is not running", should I still put them in?

File: app/controllers/admin/schools/induction_coordinators_controller.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (goodviber) [reply app/controllers/admin/schools/induction_coordinators_controller.rb:6]: `class Admin::BaseController < ApplicationController
  include Pundit

  before_action :authenticate_user!
  before_action :ensure_admin
  before_action :set_paper_trail_whodunnit
  after_action :verify_authorized
  after_action :verify_policy_scoped

private

  def ensure_admin
    raise Pundit::NotAuthorizedError, "Forbidden" unless current_user.admin?
  end
end

File: spec/cypress/integration/admin/InductionCoordinatorManagment.feature (hunk 1 above)
Comment (kicferk1) [reply spec/cypress/integration/admin/InductionCoordinatorManagment.feature:13]: Oh, when you were running it locally? Hm, I don't know what to expect locally, @callumacrae might know. 

This is branched off of something that has percy set up, right?

Comment (goodviber) [reply spec/cypress/integration/admin/InductionCoordinatorManagment.feature:13]: I will put one in and see what happens...

**Review by amazimbe — APPROVED** [review]

**Pull request merged**